contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -118,6 +143,11 @@
                 <field name="incident_polarization" type="NX_NUMBER" recommended="true" units="NX_ANY"/>
             </group>
             <group type="NXelectronanalyser">
+                <group name="model" type="NXfabrication" recommended="true">
+                    <field name="vendor" recommended="true"/>
+                    <field name="model" recommended="true"/>
+                    <field name="identifier" recommended="true"/>
+                </group>
```

Comment:
This contains the same field name "model" twice on different levels, I would suggest a different name for the NXfabrication group, e.g. device_information or vendor_information